docker.local/base.Dockerfile

```diff
@@ -10,27 +11,29 @@ RUN echo "https://mirrors.aliyun.com/alpine/v3.14/community" >> /etc/apk/reposit
 RUN echo "https://sjc.edge.kernel.org/alpine/v3.14/main" >> /etc/apk/repositories
 RUN echo "https://sjc.edge.kernel.org/alpine/v3.14/community" >> /etc/apk/repositories
 
-RUN echo "https://uk.alpinelinux.org/alpine/v3.14/main" >> /etc/apk/repositories
-RUN echo "https://uk.alpinelinux.org/alpine/v3.14/community" >> /etc/apk/repositories
+# RUN echo "https://uk.alpinelinux.org/alpine/v3.14/main" >> /etc/apk/repositories
+# RUN echo "https://uk.alpinelinux.org/alpine/v3.14/community" >> /etc/apk/repositories
 
-RUN echo "https://dl-4.alpinelinux.org/alpine/v3.14/main" >> /etc/apk/repositories
-RUN echo "https://dl-4.alpinelinux.org/alpine/v3.14/community" >> /etc/apk/repositories
+# RUN echo "https://dl-4.alpinelinux.org/alpine/v3.14/main" >> /etc/apk/repositories
+# RUN echo "https://dl-4.alpinelinux.org/alpine/v3.14/community" >> /etc/apk/repositories
 
-RUN echo "https://mirror.yandex.ru/alpine/v3.14/main" >> /etc/apk/repositories
-RUN echo "https://mirror.yandex.ru/alpine/v3.14/community" >> /etc/apk/repositories
 
 
 RUN apk add --update --no-cache build-base linux-headers git cmake bash perl grep
 
 # Install Herumi's cryptography
-RUN apk add gmp gmp-dev openssl-dev && \
-    cd /tmp && \
-    wget -O - https://github.com/herumi/mcl/archive/master.tar.gz | tar xz && \
-    wget -O - https://github.com/herumi/bls/archive/master.tar.gz | tar xz && \
-    mv mcl* mcl && \
-    mv bls* bls && \
-    make -C mcl -j $(nproc) lib/libmclbn256.so install && \
-    cp mcl/lib/libmclbn256.so /usr/local/lib && \
-    make MCL_DIR=$(pwd)/mcl -C bls -j $(nproc) install && \
-    rm -R /tmp/mcl && \
-    rm -R /tmp/bls
+RUN apk add gmp gmp-dev openssl-dev 
+
+WORKDIR /tmp
+RUN wget https://github.com/herumi/mcl/archive/master.tar.gz
+RUN tar zxvf master.tar.gz &&  mv mcl* mcl
```

Comment:
Would tar -zxvf work?

Reply:
yes, it works on my system

Reply:
yes. it works on linux/macos. our github runner is ubuntu. so it is ok.

Reply:
sorry, the shell run on alpine , not ubuntu.